### ISSUE TITLE
Wird Homematic-Wired unterstützt?

### ISSUE THREAD

**Issue by AndreasBerz** [alexreinert/debmatic#219]
Kann Debmatic auch Homematic-Wired-Geräte (nicht HomematicIP-Wired !) steuern?
Ich dazu in der Dokumentation von Debmatic nichts gefunden und gehe davon aus, dass es eher nicht funktioniert.

**Comment by alexreinert** [comment]
Doch, das geht. Man braucht natürlich das entsprechende Wired RS485 Lan Gateway (HMW-LGW-O-DR-GS-EU).

**Comment by alexreinert** [comment]
Sind noch Fragen offen?

**Comment by AndreasBerz** [comment]
Nein. Alles gut.
Ich nehme an, ich soll den issue  schließen.
